src/status_im/chat/styles/message/message.cljs

```diff
@@ -95,12 +95,11 @@
   (merge style-message-text
          {:marginTop (if incoming-group 4 0)}))
 
-(defnstyle message-view
+(defn message-view
   [{:keys [content-type outgoing group-chat selected]}]
   (merge {:padding         12
           :backgroundColor styles/color-white
```

Comment:
In a smaller PR I'd ask you to rename :camelCase keys to :kebab-case 
Maybe that's another thing to do in a blanket refactor PR

Reply:
Right sounds like a good bounty!

Reply:
https://github.com/status-im/status-react/issues/3139